docker-compose.yaml

```diff
@@ -15,33 +15,41 @@ services:
     - POSTGRES_USER=storj
     - POSTGRES_PASSWORD=storj-pass
 
-  storagenode:
-    image: storjlabs/storagenode:${VERSION}
-    environment:
-    - SATELLITE_ADDR=satellite:7777
-    - STORJ_LOG_LEVEL=info
-    links:
-    - satellite
   satellite:
     image: storjlabs/satellite:${VERSION}
     environment:
     - API_KEY=abc123
     - BOOTSTRAP_ADDR=localhost:8080
-    - IDENTITY_ADDR=:7777
-    - STORJ_MOCK_OVERLAY_NODES=INTENTIONALLY:LEFT:BLANK
-    - STORJ_LOG_LEVEL=info
+    - IDENTITY_ADDR=satellite:7777
```

Comment:
here

Reply:
I think this is ok for the local dev environment.